packages/shared/src/components/feeds/FeedNav.tsx

```diff
@@ -48,10 +70,22 @@ function FeedNav(): ReactElement {
         <Tab label={FeedNavTab.Bookmarks} url="/bookmarks" />
         <Tab label={FeedNavTab.History} url="/history" />
       </TabContainer>
-
-      <div className="fixed right-0 top-0 my-1 flex h-12 w-20 items-center justify-end bg-gradient-to-r from-transparent via-background-default via-40% to-background-default pr-4">
```

Comment:
This seem to have been available not only on mobile, and now we're replacing it with a mobile only ui. Is this intended?